tests/plugins/pastefromword/manual/pastefromword.html

```diff
@@ -0,0 +1,14 @@
+<h2>Classic editor</h2>
```

Comment:
It would be easier to verify if content was pasted correctly if the editor window was bigger. Let's set editor's height to `500` maybe?